package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@chatie/grpc",
-  "version": "0.15.2",
+  "version": "0.15.3",
```

Comment:
I think the version config in local package.json file has changed, so I don't run  `npm version patch & git push`.